masterlist.yaml

```
@@ -9836,78 +9836,50 @@ plugins:
   - name: 'Skyrimesque Counts and Countesses.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/43420' ]
     tag: [ Invent ]
-  - name: 'CNRP Brumav02.esp'
+
+  - name: 'CNRP Bruma(v02|4OOO)\.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/21388' ]
+    tag: [ Graphics ]
+  - name: 'CNRP Brumav02.esp'
     msg:
-      - type: warn
-        content:
-          - lang: en
-            text: 'Use ''CNRP Bruma4OOO.esp'' instead.'
-          - lang: de
-            text: 'Nutzen Sie stattdessen ''CNRP Bruma4OOO.esp''.'
+      - <<: *useInstead
+        subs: [ 'CNRP Bruma4OOO.esp' ]
```

`*useVersion` is a bit more appropriate in situations like these, but `*useInstead` would still work. Up to you.

```suggestion
      - <<: *useVersion
        subs: [ 'OOO/FCOM' ]
```